### ISSUE TITLE
tracktwo xcom2json question

### ISSUE THREAD

**Issue by Wizard-Toast** [Wizard-Toast/tracktwo-q#1]
@tracktwo

Hey, I'm hoping I can get a hold of you this way to ask about using your XCOM2Json tool in a project of mine.

I've written a python script that currently just reads the save jsons and displays soldier info in a way that's easy to read, but I'd like to start working on a GUI for it and turn it into more of a fully fledged save editor with a few bells and whistles.

I know in the readme you say the tool is open-source, but since there's no license on the page I just want to be absolutely sure that it would be okay with you to continue working on this and see it to a release worthy stage.

The nature of python won't let me turn it into a nice little executable file but my hope is once it's done, it'll be like a natural extension of your program.

**Comment by tracktwo** [comment]
Ah, I forgot to put in a license file in there, but it's at the top of the xcom.h header file. I usually use BSD-style but since the compression library I used is GPL so is this project.

At any rate, feel free to use it, but you said XCOM2 above, was that just a typo? Currently it supports only EU/EW.

**Comment by Wizard-Toast** [comment]
Yeah sorry, specifically I meant XCOM:EW using your xcomsave tool, with the "xcom2json.exe", "json2xcom.exe" files.

Thanks a bunch for taking the time to respond! If you're interested I'll ping you again when it's done.

**Comment by tracktwo** [comment]
Yeah, that'd be great. I was hoping it would eventually be used in some kind of easier-to-use GUI.
